docs/design/rescheduling.md

```diff
@@ -82,15 +82,14 @@ tiers:
             - name: lowPriorityFirst
             - name: lowNodeUtilization
               params:
-                nodeResourceUtilizationThresholds:
-                  thresholds:
-                    "cpu" : 20
-                    "memory": 20
-                    "pods": 20
-                  targetThresholds:
-                    "cpu" : 50
-                    "memory": 50
-                    "pods": 50
+                thresholds:
+                  "cpu" : 20
```

Comment:
@Thor-wl If i use cpu other than "cpu", does it work? it's better keep the same style with pod request.